mcpserver/args.go

```diff
@@ -0,0 +1,40 @@
+package mcpserver
+
+import "github.com/mark3labs/mcp-go/mcp"
+
+var (
+	explainationArgument = mcp.WithString("explanation",
```

Comment:
🐛 **Bug (Medium):** Typo in variable name 'explainationArgument' should be 'explanationArgument'

```suggestion
	explanationArgument = mcp.WithString("explanation",
```